### PR TITLE
ContentCard Component

### PR DESCRIPTION
**Pull request by ann-schulte-flatironschool**
ContentCard Component
Rename ContentCard -> TextCard

### CODE REVIEW

File: src/composites/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -16,10 +16,9 @@ const StyledButton = styled.button`
   cursor: pointer;
 `
 
-const StyledSidebar = styled(Grid)`
+const StyledSidebar = styled(props => <Grid item {...props} />)`
```
Comment (tmilewski) [inline src/composites/Sidebar/Sidebar.tsx:19]: I'm unsure what this is meant to accomplish. Now, per lines 79/80, no props are being passed.

File: src/components/ContentCard/ContentCard.stories.tsx [hunk 2]
```diff
@@ -7,10 +7,19 @@ export default {
...
+  <div style={{ paddingTop: '40px' }}>
+    <ContentCard {...args} />
+  </div>
+)
 
-export const Primary = Template.bind({})
-Primary.args = {
-  label: 'Full Name',
-  value: 'Johnathan Smith'
+export const Default = Template.bind({})
```
Comment (tmilewski) [inline src/components/ContentCard/ContentCard.stories.tsx:16]: Let's add a story for the JSX.Element cta. Could just be an unstyled link. Just good to have as an example.

**Review by tmilewski — CHANGES_REQUESTED** [review]

File: src/components/ContentCard/ContentCard.tsx [hunk 3]
```diff
@@ -1,38 +1,84 @@
...
+const Container = styled(props => <Grid container alignItems="center" justify="center" {...props} />)`
+  padding-bottom: 25px;
+  padding-top: 15px;
+`
+const StyledGrid = styled(props => <Grid container item xs={12} {...props} />)`
+  border: 2px solid ${Color.yellow};
+  padding: 5px;
+`
+
+const StyledLegned = styled.p`
```
[the remark this answers is not in the document]
Comment (ann-schulte-flatironschool) [reply src/components/ContentCard/ContentCard.tsx:26]: I am not sure if there is added benefit or just preference. Do you see benefit from that?

[the remark this answers is not in the document]
Comment (ann-schulte-flatironschool) [reply src/components/ContentCard/ContentCard.tsx:26]: I see your below comment now. I will try it and see if it works with getting rid of the container

File: src/composites/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (ann-schulte-flatironschool) [reply src/composites/Sidebar/Sidebar.tsx:19]: It was giving me an error and failing tests. I am not sure how it was able to get into main actually. saying the styled component was undefined. If there is a different way to fix that issue I can.

File: src/components/TextCard/TextCard.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+const StyledValue = styled.p`
+  color: ${Color.white};
+  font-size: 18px;
+  margin: 5px;
+  padding-bottom: 10px;
+  padding-left: 5px;
+`
+
+export const TextCard = ({ label, value }: TextCardProps): JSX.Element => (
+  <StyledGrid>
```
Comment (tmilewski) [inline src/components/TextCard/TextCard.tsx:30]: Let's add `{...props}` here. 1) As good practice and 2) as they're being passed in line 12.

File: src/components/ContentCard/ContentCard.tsx [hunk 5]
```diff
@@ -1,38 +1,84 @@
...
-  font-size: 18px;
-  margin: 5px;
-  padding-bottom: 10px;
-  padding-left: 5px;
+
+const StyledGridItem = styled(props => <Grid xs={12} sm={4} item {...props} />)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: ${props => (props.end ? 'flex-end' : 'flex-start')};
```
Comment (tmilewski) [inline src/components/ContentCard/ContentCard.tsx:42]: Let's use `alignItems` with values `flex-start` and `flex-end` and I'm pretty sure this won't need to be in the styles as it's built into the Grid component.

**Review by tmilewski — CHANGES_REQUESTED** [review]

**Review by tmilewski — APPROVED** [review]
🚢

**Pull request merged**